src/Symfony/Bundle/FrameworkBundle/Test/ContainerAwareTestCase.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Test;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * This class can be used as a base class for functional tests that need to
+ * access the service container.
+ *
+ * @author Matthias Pigulla <mp@webfactory.de>
+ */
+class ContainerAwareTestCase extends KernelTestCase
+{
+    /**
+     * Returns a fresh instance of the service container.
+     *
+     * Note that every invocation of this method will create a fresh container instance and also discard and
```

Comment:
Agree. Will put the change back in that keeps $container instances between invocations. But then I need to figure out why this breaks the Autowiring and CachePools tests.